src/Cameras/VR/webVRCamera.ts

```diff
@@ -140,6 +142,10 @@ export interface WebVROptions {
      */
     defaultHeight?: number;
 
+    /**
+     * If multiview should be used if availible (default: false)

```

Comment:
One thing to note is multiview is disabled by default and needs to be enabled as existing apps might expect that the output texture of the camera will not be a texture array

Reply:
Can you provide some code examples on how to use it?

Reply:
Well actually it will be in the doc so no worries for now